travis/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
OK, so this means that we're giving the whole wide world write access to https://github.com/haskell-pushbot/cabal-binaries
Guess there's no way around this in this scheme.